server/routes/testutils/testData.ts

```diff
@@ -0,0 +1,216 @@
+import { Prison } from '../../@types/bapv'
+import { SessionCapacity, SupportType, Visit } from '../../data/visitSchedulerApiTypes'
+import { InmateDetail, CaseLoad, PrisonerBookingSummary } from '../../data/prisonApiTypes'
+import { CurrentIncentive, Prisoner } from '../../data/prisonerOffenderSearchTypes'
+
+export default class TestData {
```

Comment:
Do we want to use `static` methods? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static

Reply:
Yes!